index.html

```diff
@@ -0,0 +1,1209 @@
+                    </div>
+                  </div>
+
+                  <div class="note" id="loss-of-control-mitigation" inlist="" rel="schema:hasPart" resource="#loss-of-control-mitigation">
+                    <h3 property="schema:name"><span>Note</span>: Loss of Control Mitigation</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>It is assumed that systems using WAC have mechanisms to mitigate loss of control of the resources, such as by ensuring valid ACL resources and Authorizations, by providing administrative functions on the storage, or by implicitly granting control over all resources in a storage to owners.</p>
+
+                      <p class="advisement">Source: <a href="https://github.com/solid/specification/issues/67" rel="cito:citesAsSourceDocument">issues/67</a>, <a href="https://github.com/solid/specification/issues/153" rel="cito:citesAsSourceDocument">issues/153</a>, <a href="https://github.com/solid/specification/pull/264" rel="cito:citesAsSourceDocument">pull/264</a></p>
+                    </div>
+                  </div>
+
+                  <p class="advisement">Source: <a href="https://github.com/solid/web-access-control-spec/issues/33" rel="cito:citesAsSourceDocument">issues/33</a></p>
+                </div>
+              </section>
+
+              <section id="access-subjects" inlist="" rel="schema:hasPart" resource="#access-subjects">
+                <h3 property="schema:name">Access Subjects</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>The ability of a <em>subject</em> to access a resource can be granted to identifiable agents, a class of agents, a group of agents, or an origin.</p>
+
+                  <p>The <code>acl:agent</code> predicate denotes an <a href="#agent">agent</a> being given the access permission.</p>
+
+                  <p>The <code>acl:agentClass</code> predicate denotes a <a href="#agent-class">class of agents</a> being given the access permission.</p>
+
+                  <dl>
+                    <dt id="acl-agent-class-foaf-agent"><code>foaf:Agent</code></dt>
+                    <dd>Allows access to any agent.</dd>
+                    <dt id="acl-agent-class-authenticated-agent"><code>acl:AuthenticatedAgent</code></dt>
+                    <dd>Allows access to any authenticated agent.</dd>
+                  </dl>
+
+                  <p>The <code>acl:agentGroup</code> predicate denotes a <a href="#agent-group">group of agents</a> being given the access permission. The object of an <code>acl:agentGroup</code> statement is an instance of <code>vcard:Group</code>, where the members of the group are specified with the <code>vcard:hasMember</code> predicate.</p>
+
+                  <p>The <code>acl:origin</code> predicate denotes the <a href="#origin">origin</a> an HTTP request is being given the access permission.</p>
```

Comment:
I think there's a word missing from this sentence? "the origin *of* an HTTP request" maybe?

Reply:
I think both works. Added "of". Thanks.